packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -272,30 +283,40 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
         'EncryptionPrivateKey persist to localSecondary: status $response');
     response = await _atClient?.getLocalSecondary()?.putValue(
         AT_ENCRYPTION_SELF_KEY, atKeysMap[AuthKeyType.selfEncryptionKey]!);
-    logger.finer(
-        'Self encryption key persist to localSecondary: status $response');
+    logger
+        .finer('SelfEncryptionKey persist to localSecondary: status $response');
   }
 
   @override
   Future<bool> authenticate() async {
+    // decrypts all the keys in .atKeysFile using the SelfEncryptionKey
+    // and stores the keys in a map
     var atKeysFileDataMap = await _decryptAtKeysFile(
         await _readAtKeysFile(atOnboardingPreference.atKeysFilePath));
     var pkamPrivateKey = atKeysFileDataMap[AuthKeyType.pkamPrivateKey];
 
     if (atOnboardingPreference.authMode == PkamAuthMode.keysFile &&
         pkamPrivateKey == null) {
       throw AtPrivateKeyNotFoundException(
-          'Unable to read pkam private key from provided .atKeys path: ${atOnboardingPreference.atKeysFilePath}',
+          'Unable to read PkamPrivateKey from provided atKeys file at path: '
+          '${atOnboardingPreference.atKeysFilePath}. Please provide a valid atKeys file',
           exceptionScenario: ExceptionScenario.invalidValueProvided);
     }
     await _init(atKeysFileDataMap);
-    logger.finer('pkam auth');
-    _isPkamAuthenticated = (await _atLookUp?.pkamAuthenticate())!;
-    logger.finer('pkam auth result: $_isPkamAuthenticated');
+    logger.finer('Authenticating using PKAM');
+    try {
+      _isPkamAuthenticated = (await _atLookUp?.pkamAuthenticate())!;
+    } on Exception {
```

Comment:
Exception could be due to different reasons..catch and add e.toString to the message